packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -137,6 +137,11 @@ export const DevServerMixin = {
             return res.json({ready: app.__webpackReady()})
         })
 
+        app.use('/__mrt/loading-screen/index.html', (_, res, next) => {
+            res.set('Clear-Site-Data', '"cache"')
```

Comment:
Yes, this `cache` need to be surrounded by double quotes.

Reply:
Do you wanna add a comment for it since it can be easily misplaced?